src/Files.App.Controls/Sidebar/SidebarItem.cs

```diff
@@ -277,13 +262,6 @@ private void UpdateSelectionState()
 			UpdatePointerState();
 		}
 
-		private void UpdateIcon()
-		{
-			Icon = Item?.IconSource?.CreateIconElement();
-			if (Icon is not null)
```

Comment:
Why are we no longer doing this?

Reply:
No, why are we no longer setting the UIA property appropriately? Also just because something doesnt work in xaml doesnt mean we should just ignore what the code did before.

Reply:
AccessibilityView is not a transitive property so every element needs to set it themselves. Just setting it on the presenter does not exclude its children from showing up in the UIA tree